### PR TITLE
Initial install of Docsy and configuration

### PR DESCRIPTION
**Pull request by sharifsalah**
- Copy over the following directories and files from [Docsy example](https://github.com/google/docsy-example):
  - /assets 
  - config.toml
  - /content (excluding Nordic content)
  - CONTRIBUTING.md
  - package.json
  - package-lock.json
  - .gitignore
- Add [Docsy](https://github.com/google/docsy.git) as a git submodule.
- Update config.toml for OpenCue
- Add OpenCue logo in SVG format

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (bcipriano) [inline .gitignore]: nit: I would strip out the blank lines here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```
Comment (bcipriano) [inline CONTRIBUTING.md]: Should we just point people to [the main OpenCue CONTRIBUTING file](https://github.com/imageworks/OpenCue/blob/master/CONTRIBUTING.md) here? I think we'll want to follow the same guidelines, right?

File: .gitignore (hunk 1 above)
Comment (sharifsalah) [reply .gitignore]: Thanks, and removed.

File: CONTRIBUTING.md (hunk 2 above)
Comment (sharifsalah) [reply CONTRIBUTING.md]: Good point. Actually, there doesn't seem to be much consistency between the other 'Docsy' sites. I'll update the file to ask contributors to review the main repo's guidelines. Eventually, we'll need to add some docs-specific guidelines on the opencue.io site, such as style guide...etc, but we could do that in a page on the site.

**Pull request merged**